Source/Forms/Models/Gen4/WildModel4.cpp

```diff
@@ -64,7 +64,14 @@ QVariant WildGeneratorModel4::data(const QModelIndex &index, int role) const
         case 2:
         {
             u8 item = state.getItem();
-            return item < 45 ? "No" : item < 95? "50%" : "5%";
+            if (state.getLead() == Lead::CompoundEyes)
+            {
+                return item < 20 ? "No" : item < 80? "60%" : "20%";
+            }
+            else
+            {
+                return item < 45 ? "No" : item < 95? "50%" : "5%";
+            }
```

Comment:
Maybe it would be better to do a Normal/Rare item instead of listing percents. What do you think?

Reply:
How would do this on gen5 then? You have also 1% there

Reply:
Leave the numbers then. Change `"No"` to `tr("None")`

Reply:
done